install/scripts/php-conf-fix.sh

```diff
@@ -6,7 +6,24 @@
 #
 echo 'Automated php.ini configuration adjustments'
 echo
-phpIni=/etc/php5/apache2/php.ini
+if [ -f /etc/debian_version ]
```

Comment:
At the moment there is no default case, i.e. `phpIni` might be empty.

I would recommend to update the code as follows:
```bash
if [ -f /etc/redhat-release ]; then
    phpIni=/etc/php.ini
    TIMEZONE=`readlink /etc/localtime | sed 's%/usr/share/zoneinfo/%%'`
else
    phpIni=/etc/php5/apache2/php.ini
    TIMEZONE=`cat /etc/timezone`
fi
if [ -z $TIMEZONE ]; then
    TIMEZONE="America/Denver"
fi
```